cmake_modules/commonSetup.cmake

```diff
@@ -739,4 +739,16 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
         endif()
   ENDMACRO()
 
+  ###########################################################################
+  ###
+  ## The following sets the dependency list for a package
+  ###
+  MACRO(SET_DEPENDENCIES dependencies cpackvar)
```

Comment:
great job - minor question: can we reorder so cpackvar is first, to be more like it was ?